cmd/kyma/deploy/cmd.go

```diff
@@ -0,0 +1,669 @@
+  - To tweak the values on a component level, use deploy --components:
+    Pass a components list that includes only the components you want to test
+    and try out the settings that work for your installation.
+  - To understand which component failed during deployment, deactivate the default atomic deployment:
+		--atomic=false
+    With atomic deployment active, any component that hasn't been installed successfully is rolled back,
+    which may make it hard to find out what went wrong. By disabling the flag, the failed components are not rolled back.
+	`,
+		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases: []string{"d"},
+	}
+
+	// default value for workspace flag is set in validateFlags()
+	// to avoid having the actual home directory shown in the auto-generated docs
+	cobraCmd.Flags().StringVarP(&o.WorkspacePath, "workspace", "w", "", `Path to download Kyma sources (default "$HOME/.kyma/sources" or ".kyma-sources")`)
+	cobraCmd.Flags().BoolVarP(&o.Atomic, "atomic", "a", false, "Set --atomic=true to use atomic deployment, which rolls back any component that could not be installed successfully.")
+	// default value for components-file flag is set in validateFlags()
+	// to avoid having the actual home directory shown in the auto-generated docs
+	cobraCmd.Flags().StringVarP(&o.ComponentsFile, "components-file", "c", "", `Path to the components file (default "$HOME/.kyma/sources/installation/resources/components.yaml" or ".kyma-sources/installation/resources/components.yaml")`)
+	cobraCmd.Flags().StringSliceVarP(&o.Components, "component", "", []string{}, "Provide one or more components to deploy (e.g. --component componentName@namespace)")
+	cobraCmd.Flags().StringSliceVarP(&o.OverridesFiles, "values-file", "f", []string{}, "Path(s) to one or more JSON or YAML files with configuration values")
+	cobraCmd.Flags().StringSliceVarP(&o.Overrides, "value", "", []string{}, "Set configuration values. Can specify one or more values, also as a comma-separated list (e.g. --value component.a='1' --value component.b='2' or --value component.a='1',component.b='2').")
+	cobraCmd.Flags().DurationVarP(&o.Timeout, "timeout", "", 20*time.Minute, "Maximum time for the deployment")
+	cobraCmd.Flags().DurationVarP(&o.TimeoutComponent, "timeout-component", "", 6*time.Minute, "Maximum time to deploy the component")
+	cobraCmd.Flags().IntVar(&o.Concurrency, "concurrency", 4, "Number of parallel processes")
+	cobraCmd.Flags().StringVarP(&o.Domain, "domain", "d", "", "Custom domain used for installation")
+	cobraCmd.Flags().StringVarP(&o.TLSCrtFile, "tls-crt", "", "", "TLS certificate file for the domain used for installation")
+	cobraCmd.Flags().StringVarP(&o.TLSKeyFile, "tls-key", "", "", "TLS key file for the domain used for installation")
+	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", defaultSource, `Installation source:
+	- Deploy a specific release, for example: "kyma deploy --source=1.17.1"
```

Comment:
When alpha commands are stable, we'll have k2.0. 
 
```suggestion
	- Deploy a specific release, for example: "kyma deploy --source=2.0.0"
```